packages/clay-css/gulpfile.js

```diff
@@ -194,6 +195,19 @@ gulp.task(
 	}
 );
 
+gulp.task('copy:licenses', function() {
+	var newLicenseDir = path.join('.', 'LICENSES');
+	var originalLicenseDir = path.join('..', '..', 'LICENSES');
+
+	return gulp.src([
+		path.join(originalLicenseDir, 'BSD-3-Clause.txt'),
+		path.join(originalLicenseDir, 'LicenseRef-MIT-Bootstrap.txt'),
+		path.join(originalLicenseDir, 'MIT.txt')
```

Comment:
Would a glob work here? If so, it would be more future-proof (ie. if we ever add/remove licenses):

```js
path.join(originalLicenseDir, '*.txt')
```

Reply:
I was also trying to future-proof since all of our licenses for Clay would live in this directory. I didn't want to include licenses for React or other module we might be using in our React components.